tests/Configurator/EnvConfiguratorTest.php

```diff
@@ -45,6 +45,7 @@ public function testConfigure()
             '#1' => 'Comment 1',
             '#2' => 'Comment 3',
             'APP_SECRET' => 's3cretf0rt3st',
+            'DATABASE_URL' => '"mysql://test"',
```

Comment:
I noticed (and updated my issue) that `&` was also not escaped and breaks the phpunit config.
A test environment variable could be something like this `"mysql://test/symfony?foo=<bar>&version=1.0"` to ensure that all characters are escaped.

What do you think ?